keyboards/idobo/keymaps/drewdobo/keymap.c

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2020 Drew Smathers
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+// Keyboard Layers
+#define _QW 0
+#define _QW_MAC 1
+#define _FN 2
+#define _MOUSE 3
+
+// Some short-cut aliases
+#define Z_LALT LALT_T(KC_ESC)
+#define Z_MAC DF(_QW_MAC)
+#define Z_PC DF(_QW)
+#define ___N___ KC_NO
```

Comment:
QMK defines an alias for this keycode automatically for every keymap.

https://github.com/qmk/qmk_firmware/blob/140c08e521084410ccb239848a3253b13c12683d/quantum/quantum_keycodes.h#L31

Reply:
Update to use alias already defined in QMK.